tests/test_device_manager.py

```diff
@@ -645,7 +648,7 @@ def test_get_available_devices(self, device_manager: DeviceManager) -> None:
         assert afg.query("*IDN?") == "TEKTRONIX,AFG3252C,SERIAL1,SCPI:99.0 FV:3.2.3"
         assert available_devices["local"] == device_tuple
         assert available_devices["configured"] == (
-            "address=AFG3252C-HOSTNAME,connection_type=TCPIP,device_type=AFG",
+            "address=AFG3252C-HOSTNAME,connection_type=TCPIP,device_type=AFG,lan_device_name=inst0",
```

Comment:
I am not sure if this should be here, If nothing was configured, this should be empty (since it is the default).

Reply:
This is needed since the connection type is TCPIP and no LAN device name is provided, the DeviceConfigEntry defaults to using 'inst0'

Reply:
Look at how some of the other dataclasses handle it. We shouldn't be putting out more information in the config printout if the user isn't explicitly setting the `lan_device_name`. If the `lan_device_name` is unset, it should default to `'inst0'`, but nothing extra should be printed out when the config is printed out.

Reply:
The other helpers can default to `'inst0'` if nothing is specified by the dataclass.

Reply:
Moved default `lan_device_name` to `inst0` from post init to `get_visa_resource_expression` function. Moved default `gpib_board_number` to 0 to match implementation.